src/front/index.scss

```diff
@@ -1,17 +1,50 @@
+/* Mixins & functions */
+@import 'app/common/styles/mixins';
+
+/* 3rd-party */
 @import '../../node_modules/normalize.css/normalize.css';
+
+/* Common components */
+@import 'app/common/components/iconButton/index';
+
+/* Components */
 @import 'app/features/todo/todo-screen';
 @import 'app/features/login/login-screen';
 @import './app/features/navigation/navigation';
 @import 'app/features/challenges/components/index';
 
+
+
 * {
-  box-sizing: border-box;
+    box-sizing: border-box;
 }
 
 .icon {
```

Comment:
right) loop and modifications description inside index.scss, and not in some shared used scss file for instance which will be included in index.scss as a partial as above